build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -60,8 +60,9 @@ featureGates:
 {{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "Multicluster" "default" false) }}
 
 # Enable support for provisioning secondary network interfaces for Pods (using
-# Pod annotations). At the moment, Antrea can only create secondary network
-# interfaces using SR-IOV VFs on baremetal Nodes.
+# Pod annotations). At the moment, Antrea can create secondary network
+# interfaces either using SR-IOV VFs on bare-metal Nodes or veth interfaces
+# bridged to the underlay network, with or without VLAN tagging.
```

Comment:
What does this sentence "veth interfaces bridged to the underlay network, with or without VLAN tagging" mean?

Reply:
I checked the implementation of VLAN type and think the secondary network is a veth device connected to an OVS bridge with a pNIC attached, so isn't it essentially "Antrea can create secondary network interfaces using veth interfaces bridged to the underlay network"? "with or without VLAN tagging" means traffic from secondary network interfaces can support both VLAN tagging and non VLAN tagging.

Reply:
Ah, my bad - I thought the sentence is about "Antrea cannot"..